src/libraries/Microsoft.Extensions.Logging.Console/src/SimpleConsoleFormatter.cs

```diff
@@ -0,0 +1,247 @@
+
+        public override void Write<TState>(in LogEntry<TState> logEntry, IExternalScopeProvider scopeProvider, TextWriter textWriter)
+        {
+            LogLevel logLevel = logEntry.LogLevel;
+            ConsoleColors logLevelColors = GetLogLevelConsoleColors(logLevel);
+            string logLevelString = GetLogLevelString(logLevel);
+
+            string timestamp = null;
+            string timestampFormat = FormatterOptions.TimestampFormat;
+            if (timestampFormat != null)
+            {
+                DateTime dateTime = GetCurrentDateTime();
+                timestamp = dateTime.ToString(timestampFormat);
+            }
+            if (timestamp != null)
+            {
+                textWriter.Write(FormatterOptions.SingleLine ? timestamp + ' ' : timestamp);
+            }
+
+            if (logLevelString != null)
+            {
+                textWriter.WriteColoredMessage(logLevelString, logLevelColors, FormatterOptions.DisableColors);
+            }
+
+            StringBuilder logBuilder = _logBuilder;
+            _logBuilder = null;
+
+            if (logBuilder == null)
+            {
+                logBuilder = new StringBuilder();
+            }
+
+            CreateDefaultLogMessage(logBuilder, logEntry, scopeProvider);
+            textWriter.Write(logBuilder.ToString());
+
+            logBuilder.Clear();
+            if (logBuilder.Capacity > 1024)
+            {
+                logBuilder.Capacity = 1024;
+            }
+            _logBuilder = logBuilder;
+        }
+
+        private void CreateDefaultLogMessage<TState>(StringBuilder logBuilder, in LogEntry<TState> logEntry, IExternalScopeProvider scopeProvider)
+        {
+            bool singleLine = FormatterOptions.SingleLine;
+            string category = logEntry.Category;
+            int eventId = logEntry.EventId.Id;
+            Exception exception = logEntry.Exception;
+            string message = logEntry.Formatter(logEntry.State, exception);
+
+            // Example:
+            // info: ConsoleApp.Program[10]
+            //       Request received
+
+            // category and event id
+            logBuilder.Append(LoglevelPadding + category + '[' + eventId + "]");
+            if (!singleLine)
+            {
+                logBuilder.Append(Environment.NewLine);
+            }
+
+            // scope information
+            GetScopeInformation(logBuilder, scopeProvider, singleLine);
+            if (singleLine)
+            {
+                logBuilder.Append(' ');
+            }
+            WriteMessage(logBuilder, message, singleLine);
+
+            // Example:
+            // System.InvalidOperationException
+            //    at Namespace.Class.Function() in File:line X
+            if (exception != null)
+            {
+                // exception message
+                WriteMessage(logBuilder, exception.ToString(), singleLine);
+            }
+            if (singleLine)
```

Comment:
>singleLine [](start = 16, length = 10)

this should be !singleline?

Reply:
No. When doing a singleLine simple log, then only at the end we write Environment.NewLine as you see here.
Otherwise, for multiline, earlier exception and message content were written into a separate line.